test/unit/gateways/paymentez_test.rb

```diff
@@ -157,6 +157,7 @@ def test_partial_refund
       assert_match(/"amount":1.0/, data)
     end.respond_with(successful_refund_response)
     assert_success response
+    assert_not_nil response.message
```

Comment:
I think it's worth checking the actual content of the message field throughout the unit tests since we're specifying the responses.

```suggestion
    assert_equal 'Completed', response.message
```